src/usr/local/www/firewall_rules_edit.php

```diff
@@ -1216,6 +1258,7 @@ function build_if_list() {
 	'Protocol',
 	$pconfig['proto'],
 	array(
+		'any' => gettext('any'),
```

Comment:
Points taken - can you take a look at an alternate fix though [(8afd8c0)](https://github.com/stilez/pfsense-stilez-working/commit/8afd8c0409a9ed246a244e7a1dbdcb2ad627434b) which seems to meet all requirements neatly, and let me know if it is acceptable for resolving your point?